learnify/mobile/assets/lang/en.json

```diff
@@ -49,5 +49,10 @@
   "events": "Events",
   "forum": "Forum",
   "create_chapter": "Create a Chapter",
-  "edit_chapter": "Edit the Chapter"
+  "edit_chapter": "Edit the Chapter",
```

Comment:
Since there will be no sequential structure in the course because of the co-learning, we changed the "chapter" term with "post". We can change it here as well.

Reply:
We can do this small change in another opened branch which is related to edit chapter.